modules/ROOT/pages/using-opscenter.adoc

```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
 After logging into Ops Center, you'll see a list of tabs on the left of the page. Navigate to "Monitoring"
 to view the dashboards available for viewing.
 
-=== View Metrics from an Application
+The dashboards are divided into the following categories:
+
+* *Applications* shows metrics on all the replicas for an application deployment. 
+** Filters include the environment ID the application is deployed in, and the application deployment name.
+* *Cluster* shows metrics on the overall Runtime Fabric cluster and provides a machine-level overview of the Kubernetes cluster. The limit for a node is the sum of all limits for individual pods scheduled on it. Unbound pods, like ingress, are not included in the limit, as they are not limited. This dashboard will not show ingress-specific data but machine-level data only, including that all pods. 
```

Comment:
Suggestion:
his dashboard shows only machine-level data for all pods.
or
This dashboard does not show ingress-specific data, but does show machine-level data for all pods.